src/Infrastructure/Data/DataTransferObject.php

```diff
@@ -77,7 +77,7 @@ abstract public function toArray();
      *
      * @return mixed
      */
-    protected static function getDataValue(array $rawData, string $key, mixed $default = '')
+    protected static function getDataValue(array $rawData, string $key, $default = '')
```

Comment:
Did you remove the 'mixed' here intentionally?

Reply:
Yes, unit tests were failing